packages/gitmoji-changelog-core/src/index.js

```diff
@@ -84,7 +92,14 @@ function sortVersions(c1, c2) {
 
 function hasNextVersion(tags, release) {
   if (!release || release === 'next') return true
-  return tags.some(tag => semver.eq(tag, release))
+  try {
+    return tags.some(tag => semver.eq(tag, release))
+  } catch (e) {
+    if (e.name === 'TypeError') {
+      throw FunctionalError(e.message)
```

Comment:
It's hard to know what kind of functional error it is. I think you should write a specific error message to  explain the error. It will help to understand the code and handled errors.